### PR TITLE
Nethunuri java

### PR DESCRIPTION
**Pull request by nethunuri**

### CODE REVIEW

File: .gitignore
Comment (sujith-es) [inline .gitignore:1]: Why is this file modified?

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+plugins {
+    // Apply the Java Gradle plugin development plugin to add support for developing Gradle plugins
+    `java-gradle-plugin`
+}
+
+repositories {
+    // Use jcenter for resolving dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    jcenter()
```
Comment (sujith-es) [inline build.gradle.kts]: `jCenter` is deprecated. Could you add it's replacement?

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+dependencies {
+    // Use JUnit test framework for unit tests
+    testImplementation("junit:junit:4.12")
+}
+
+gradlePlugin {
```
Comment (sujith-es) [inline build.gradle.kts:23]: No need to write `Task` in Plugin block. There are in-build features available.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-bin.zip
```
Comment (sujith-es) [inline gradle/wrapper/gradle-wrapper.properties]: Why gradle 6.0 version?  Latest version is 8.3

File: src/test/java/com/hello/HelloWorldTest.java [hunk 4]
```diff
@@ -2,7 +2,6 @@
 
 import static org.junit.Assert.assertEquals;
 
-import com.hello.HelloWorld;
```
Comment (sujith-es) [inline src/test/java/com/hello/HelloWorldTest.java]: Why delete the import?

**Review by sujith-es — CHANGES_REQUESTED** [review]

**Comment by nethunuri** [comment]
Gradle Upgraded to 8.7 and updated with changes.

**Review by sujith-es — APPROVED** [review]
👍
